wger/exercises/models.py

```diff
@@ -330,8 +331,16 @@ def set_author(self, request):
         if request.user.has_perm('exercises.add_exercise'):
             self.status = self.STATUS_ACCEPTED
             if not self.license_author:
-                self.license_author = 'wger.de'
+                #  Get everything up to semicolon, exclusive.
+                #  Example: my-gym.com:9090 returns my-gym.com
+                results = re.search('^([^:]+).*', request.get_host())
+
+                if results:
+                    name = results.group(1)
+                else:
+                    name = request.get_host()
```

Comment:
Regexes are a bit of overkill here, we can easily extract the part from the hostname that we need with a split: ``get_host().split(':')[0]``